baker/formatWordpressPost.tsx

```diff
@@ -540,9 +541,21 @@ export const formatWordpressPost = async (
         $heading.append(`<a class="${DEEP_LINK_CLASS}" href="#${slug}"></a>`)
     })
 
+    // Extracting the useful information from the HTML
+    const stickyNavLinks: { text: string; target: string }[] = []
+    const $stickyNavContents = cheerioEl(".sticky-nav-contents")
+    const $stickyNavLinks = $stickyNavContents.children().children()
+    $stickyNavLinks.each((_, element) => {
+        const text = get(element, ["children", 0, "children", 0, "data"])
+        const target = get(element, ["attribs", "href"])
```

Comment:
I think you can do without the use of `get` here, especially seeing that this is a cheerio element (so, basically jQuery).